chapter2/tests/t20/terraform/dev/alb.tf

```diff
@@ -0,0 +1,45 @@
+module "alb" {
+  source             = "terraform-aws-modules/alb/aws"
+  version            = "~> 8.6.1"
+  name               = local.labels.alb
+  load_balancer_type = "application"
+  vpc_id             = data.aws_vpc.target.id
+  subnets            = data.aws_subnets.wordpress.ids
+  security_groups    = [module.wordpress_alb_sg.security_group_id]
+  tags               = var.tags
+
+  target_groups = [
+    {
+      name             = local.labels.tg
+      backend_protocol = "HTTP"
+      backend_port     = 80
+      targets          = aws_alb_target_group_attachment.instance_attachments
+    }
+  ]
+
+  https_listeners = [
+    {
+      port               = 443
+      protocol           = "HTTPS"
+      target_group_index = 0
+      ssl_policy         = "ELBSecurityPolicy-2016-08"
+      certificate_arn    = aws_acm_certificate.tf-maxim-omelchenko.arn
+    }
+  ]
+  depends_on = [module.ec2_instances, module.wordpress_alb_sg, aws_acm_certificate_validation.cert]
```

Comment:
make multiline when you have long depends_on >=3 items

Reply:
> make multiline when you have long depends_on >=3 items

@dmitry-mightydevops
Done